duckduckgo/Android
# Update libraries

Task/Issue URL: https://app.asana.com/0/414730916066338/1200541903185345/f
Tech Design URL: 
CC: 

**Description**:
This PR updates the following libraries:

- version.androidx.room: 2.2.6 -> 2.3.0
- version.kotlin=1.4.10 -> 1.5.20
- version.androidx.appcompat=1.2.0 -> 1.3.0
- version.androidx.webkit=1.3.0 -> 1.4.0
- version.androidx.core=1.3.2 → 1.5.0
- version.androidx.fragment=1.2.5 → 1.3.5
- version.androidx.work=2.4.0 -> 2.5.0
- version.okhttp3=4.9.0 -> 4.9.1
- version.mockito=3.4.6 ->3.10.0

**Steps to test this PR**:
1. App should build and pass all the tests.
1. Install the app and perform any action to ensure no unexpected issues.

Smoke test:
- Onboarding CTAs
- Add bookmarks and favorites
- disable/enable privacy protection
- share feedback
- fireproof a site and clear all data
- opening multiple tabs (and closing tabs + undo)

Note: All the changes have been tested already on API 21

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
